capdl.mk

```diff
@@ -57,14 +57,14 @@ specs: $(SEL4_SPECS) $(SEL4_MODULE_SPECS)
 
```

Comment:
Is this file still used? AFAIR this was support for the version of the CapDL initialiser that handled device specs. Devices are now fully supported in CapDL specs and by the main initialiser. Can someone who knows more of the history of this file comment?